doc/releases/changelog-0.30.0.md

```diff
@@ -1,146 +1,285 @@
-  [(#3759)](https://github.com/PennyLaneAI/pennylane/pull/3759)
+  ```pycon
+  >>> circuit(0.5)
+  tensor([0.87758256, 0.93879128, 0.06120872], requires_grad=True)
+  ```
 
-* The `apply_operation` function added to `devices/qubit` now supports broadcasting.
-  [(#3852)](https://github.com/PennyLaneAI/pennylane/pull/3852)
+  In versions 0.30 and above, `circuit()` returns a length-2 tuple containing the expectation
+  value and probabilities separately:
 
-* `pennylane.devices.qubit.preprocess` now allows circuits with non-commuting observables.
-  [(#3857)](https://github.com/PennyLaneAI/pennylane/pull/3857)
+  ```pycon
+  >>> circuit(0.5)
+  (tensor(0.87758256, requires_grad=True),
+   tensor([0.93879128, 0.06120872], requires_grad=True))
+  ```
 
-* Adjoint differentiation support for the new qubit state-vector device has been added via
-  `adjoint_jacobian` in `devices/qubit`.
-  [(#3790)](https://github.com/PennyLaneAI/pennylane/pull/3790)
+  More details about this change, along with help and troubleshooting tips to solve any issues,
+  can be found [here](https://docs.pennylane.ai/en/stable/introduction/returns.html).
 
-* `qml.devices.qubit.measure` now computes the expectation values of `Hamiltonian` and `Sum`
-  in a backpropagation-compatible way.
-  [(#3862)](https://github.com/PennyLaneAI/pennylane/pull/3862/)
+<h4>A bunch of performance tweaks 🏃💨</h4>
 
-* `MeasurementProcess.shape` now accepts a `Shots` object as one of its arguments to reduce exposure to unnecessary
-  execution details.
-  [(#4012)](https://github.com/PennyLaneAI/pennylane/pull/4012)
+* Single-qubit operations that have multi-qubit control can now be decomposed more efficiently
+  using fewer CNOT gates.
+  [(#3851)](https://github.com/PennyLaneAI/pennylane/pull/3851)
 
-<h4>Performance improvements</h4>
+  Three decompositions from [arXiv:2302.06377](https://arxiv.org/abs/2302.06377) are provided and
+  compare favourably to the already-available ZYZ decomposition:
 
-* Hardware-compatible pulse sequence gradients with `stoch_pulse_grad` can be calculated faster now, using
-  the new keyword argument `use_broadcasting`. Executing a `ParametrizedEvolution` that returns
-  intermediate evolutions has increased performance as well, using the state vector ODE solver.
-  [(#4000)](https://github.com/PennyLaneAI/pennylane/pull/4000)
-  [(#4004)](https://github.com/PennyLaneAI/pennylane/pull/4004)
+  ```python
+  wires = [0, 1, 2, 3, 4, 5]
+  control_wires = wires[1:]
 
-* Added a new decomposition to `qml.SingleExcitation` that halves the number of
-  CNOTs required.
-  [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
+  op = qml.RX(np.pi / 2, wires=0)
 
-* Improved efficiency of `tapering()`, `tapering_hf()` and `clifford()`.
-  [(3942)](https://github.com/PennyLaneAI/pennylane/pull/3942)
+  with qml.tape.QuantumTape() as tape:
+      qml.ctrl(op, control=control_wires)
+  
+  with qml.tape.QuantumTape() as zyz_tape:
```

Comment:
out of curiosity, is there a reason for using tapes and `.specs` here, instead of QNodes and `qml.specs`?

Reply:
I couldn't get it to work with `qml.specs` :disappointed:

At the QNode level, it's too limited in terms of users controlling the expansion. For example:
```python
dev = qml.device("null.qubit", wires=wires)

@qml.qnode(dev)
def f():
    qml.ctrl(op, control=control_wires)
    return qml.expval(qml.PauliZ(0))

qml.specs(f, expansion_strategy="device", max_expansion=10)()["resources"].gate_types
```
The result is `defaultdict(int, {'Hadamard': 2, 'MultiControlledX': 4, 'QubitUnitary': 4})` because all of those gates are supported on the device.

Reply:
:(